linera-base/src/identifiers.rs

```diff
@@ -178,6 +178,27 @@ impl<A> Debug for BytecodeId<A> {
     }
 }
 
+impl<A> Display for BytecodeId<A> {
```

Comment:
We want `Display` to print the BCS-bytes as hexadecimal. I think the `bcs_scalar!` macro already takes care of that.